packages/swimlane/data_stream/swimlane_api/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,197 @@
+---
+description: Pipeline for Swimlane API Logs
+processors:
+    - set:
+        field: ecs.version
+        tag: set_ecs_version
+        value: 8.17.0
+    - set:
+        field: event.original
+        copy_from: message
+    - json:
+        field: message
+        target_field: json
+        ignore_failure: true
+    - drop:
+        if: ctx.json == null || !(ctx.json instanceof Map)
+    - remove:
+        field: 
+          - message
+          - input
+          - stream
+          - log
+        ignore_missing: true
+    - rename:
+        field: json
+        target_field: swimlane.audit_log
+        ignore_missing: true
+    - drop:
+        if: 'ctx.swimlane.audit_log.LogType != "Audit"'
```

```suggestion
        if: ctx.swimlane?.audit_log?.LogType != "Audit"
```
(similar below: quoting is only needed for strings starting with chars that are otherwise meaningful to YAML and we should use null-safe operators generally unless locally provable to not be neede)